app/views/admin/shared/_admin_nav.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="container">
+  <h3>Admin Dashboard</h3>
+  <nav class="nav nav-tabs" id="myTab" role="tablist">
+    <a class="nav-item nav-link active" id="nav-home-tab" data-toggle="tab" href="#nav-home" role="tab" aria-controls="nav-home" aria-expanded="true">View Orders</a>
+    <%= link_to "View Items", admin_items_path, class: "nav-item nav-link" %>
+    <%= link_to "Update Account", edit_user_path(current_user), class: "nav-item nav-link" %>
+    <%= link_to "View Analytics", admin_analytics_path, class: "nav-item nav-link" %>
+    <%= link_to "Stores", admin_stores_path, class: "nav-item nav-link" %>
```

Comment:
I've been thinking about this "Stores" / "My Stores" link for a bit. 

Since "Stores" is only used for changing the status of stores (pending to active, etc) that only applies to Platform Admins, how do y'all feel about changing the link name and putting a conditional on it so that it displays only if the current_user is a platform admin?

<%= link_to "Approve/Suspend Stores", admin_stores_path, class: "nav-item nav-link" if current_user.platform_admin? %>

Reply:
Agreed! Good catch! I'll put that in now